test_runner/regress/test_compute_migrations.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING, cast
+
+import pytest
+from fixtures.compute_migrations import NUM_COMPUTE_MIGRATIONS
+
+if TYPE_CHECKING:
+    from fixtures.neon_fixtures import NeonEnv
+
+
+MIGRATIONS: tuple[str, ...] = (
+    "0001-neon_superuser_bypass_rls",
```

Comment:
Maybe just figure this out by listing the `COMPUTE_MIGRATIONS_DIR` + trimming the `.sql`? It's always a pain to maintain some lists in both places